temba/context_processors_weni.py

```diff
@@ -1,12 +1,21 @@
 from django.conf import settings
 
 
-def enable_weni_layout(request):
+def use_weni_layout(request):
 
     host = request.get_host().split(":")[0]
 
     return {"use_weni_layout": host.endswith(settings.WENI_DOMAINS["weni"])}
 
+def show_sidemenu(request):
+    if request.path == '/':
+        return {"show_sidemenu": False}
+
+    for p in settings.SIDEBAR_EXCLUDE_PATHS:
```

Comment:
what is `p`? is `path`? if `p` is `path`, change the name variable `p` to `path` please.